db/system_keyspace.cc

```diff
@@ -2321,6 +2321,9 @@ future<> system_keyspace::make(
         co_await db.create_local_system_table(table, maybe_write_in_user_memory(table), erm_factory);
         co_await db.find_column_family(table).init_storage();
     }
+    if (db.get_config().enable_tablets()) {
```

Comment:
Since 4b21cca443270280b87c3dbad52855cc2867ec5d the tablets feature is always enabled, and this config option just determines the default for new keyspaces, so the types should be added based on the cluster feature instead of the config option.

Reply:
This is in the very early boot stage. Can we actually use the cluster feature?

Reply:
The enabled features are loaded only in on_system_tables_loaded which is called after init_system_keyspace

Reply:
OK. I will drop the check since it is useless now.